app/Core/Theme.php

```diff
@@ -751,14 +752,16 @@ public function setSchemeColors($colorscheme)
 
         if (isset($this->colorSchemes[$colorscheme]["primaryColor"])) {
             $primary = $this->colorSchemes[$colorscheme]["primaryColor"];
-            $_SESSION["usersettings.colorScheme.primaryColor"] = $primary;
         }
 
         if (isset($this->colorSchemes[$colorscheme]["secondaryColor"])) {
             $secondary = $this->colorSchemes[$colorscheme]["secondaryColor"];
-            $_SESSION["usersettings.colorScheme.secondaryColor"] = $secondary;
         }
 
+        $_SESSION["usersettings.colorScheme.primaryColor"] = $primary;
+        $_SESSION["usersettings.colorScheme.secondaryColor"] = $secondary;
+
+
```

Comment:
I think we have some merge conflicts issues. I fixed this before too so that it can't throw an error.

Reply:
yeah that must have been a bad merge. Reverting. The change important here was line 342, and just setting an id if it can't find the theme. Theme is not so crucial that a failure to load the theme should stop the application. We just revert to the default theme.